packages/orchestration/README.md

```diff
@@ -413,12 +415,12 @@ Both `chatCompletion()` and `getContent()` methods can throw errors.
 
 - **Axios Errors**:  
   When the chat completion request fails with a `400` status code, the caught error will be an `Axios` error.
-  The property `error.response.data.message` provides additional details about the failure.
+  Check `error.cause.response.data` for the error response body.
```

Comment:
Is this the best way to handle an error with ErrorWithCause, or can a property of the new Error be used instead?

Reply:
Unfortunately no. Because `ErrorWithCause` is meant to be generic also for errors that are not `AxiosError`. So user has to access the response data by themselves in case it comes from Axios. But at least we print the response for them in the stack already.